cmake/package_deb.cmake

```diff
@@ -74,7 +74,7 @@ set(CPACK_COMPONENTS_ALL applications doc library share ${THIRDPARTY_COMPONENT_G
 ## Autogeneration with SHLIBDEPS will add to this variable. For now we include most things statically and require the standard Qt package and libc6 only.
 ## (only available in Ubuntu >=17.10). For older Ubuntu, dependencies can be installed from a thirdparty repo.
 set(CPACK_DEBIAN_PACKAGE_DEPENDS 
-  "qtbase5-dev (>= 5.15.3), libqt5svg5 (>= 5.15.3), libc6 (>= 2.28), libqt5widgets5 (>= 5.15.3), libqt5gui5 (>= 5.15.3), libqt5core5 (>= 5.15.3) | libqt5core5t64 (>= 5.15.3)")
+  "qt6base-dev (>= 6.2.2), libqt6svg6 (>= 6.2.2), libc6 (>= 2.28), libqt6widgets6t64 (>= 6.2.2), libqt6gui6t64 (>= 6.2.2), libqt6core6t64 (>= 6.2.2)")
```

Comment:
I thought the t64 packages are only for debian-unstable. I would recommend to build a package that supports debian-stable instead.

Reply:
You can add ( x | y ) to indicate that either of them has to be found.

Reply:
Good catch. It looks like 24.04 and 24.10 include the t64 version and 22.04 use the regular one. I'll add the | logic.